src/databricks/labs/ucx/queries/assessment/main/00_7_count_storage_credentials.sql

```diff
@@ -0,0 +1,5 @@
+/* --title 'Total Storage Credentials'
+--overrides '{"spec":{"encodings":{"value":{"fieldName":"count_total_storage_credentials","rowNumber":2,"style":{"rules":[{"condition":{"operator":">","operand":{"type":"data-value","value":"200"}},"color":"#E92828"}]},"displayName":"count_total_storage_credentials"}}}}' */
```

Comment:
Maybe format this a bit:
```suggestion
/* 
--title 'Total Storage Credentials'
--overrides '{"spec": {"encodings": {"value": {
    "fieldName": "count_total_storage_credentials",
    "rowNumber": 2,
    "style": {"rules": [{"condition": {
        "operator": ">", 
        "operand": {"type": "data-value", "value": "200"}}, 
        "color": "#E92828"
    }]},
    "displayName":"count_total_storage_credentials"}}}}' 
*/
```